src_c/bitmask.c

```diff
@@ -39,7 +39,8 @@
 static INLINE unsigned int
 bitcount(BITMASK_W n)
 {
-    if (BITMASK_W_LEN == (32)) {
+    const int bitmask_len = BITMASK_W_LEN;
```

Comment:
I'm not entirely sure why this wasn't/isn't an `#if BITMASK_W_LEN == 32`?

Reply:
This was weird to me too at start. But then I figured out that `BITMASK_W_LEN` uses `sizeof` which, weirdly enough, is not a macro. So you can't really use preprocessor macros with it, and if you use it in regular `if` statement checks, then compiler warns about constant comparison. To make the compiler happy I had to store the value first in a variable and then do regular comparisons with it